### PR TITLE
change of maven builder configuration

### PR DESCRIPTION
**Pull request by allaVolkov**
Before submitting a pull request, please ensure the following:

## Description

Please explain the changes you made.

Add a link to the design (if applicable).

### Checklist
- [x] Code compiles correctly
- [x] Relevant tests were added (unit / contract / integration)
- [x] Relevant logs were added
- [x] Formatting and linting run locally successfully
- [x] All tests pass
- [ ] UA review
- [ ] Design is documented
- [ ] Extended the README / documentation, if necessary
- [NA] Open source is approved

### CODE REVIEW

File: internal/commands/deprecated_builders.go [hunk 1]
```diff
@@ -0,0 +1,13 @@
+package commands
+
+import "github.com/SAP/cloud-mta-build-tool/internal/logs"
+
+// List of deprecated builders
+var deprecatedBuilders = map[string]string{"maven_deprecated": `the "maven_deprecated" builder is deprecated and will be removed after July, 2021.`}
+
+func awareOfDeprecatedBuilder(builder string) {
```
Comment (tal-sapan) [inline internal/commands/deprecated_builders.go:8]: Maybe rename to `warnIfDeprecatedBuilder`

**Review by tal-sapan — COMMENTED** [review]
I will approve after the "what's new" document has content

File: internal/commands/deprecated_builders.go (hunk 1 above)
Comment (allaVolkov) [reply internal/commands/deprecated_builders.go:8]: thought about this, i wanted to be less concrete

Comment (allaVolkov) [reply internal/commands/deprecated_builders.go]: done

File: docs/docs/whatsnew.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
+#<b>Important updates</b>
```
Comment (tal-sapan) [inline docs/docs/whatsnew.md:1]: Add a space after the `#`
```suggestion
# <b>Important updates</b>
```

File: docs/docs/whatsnew.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+##v1.1.0
```
Comment (tal-sapan) [inline docs/docs/whatsnew.md:3]: Add a space after the `#`
```suggestion
## v1.1.0 
```

File: docs/docs/whatsnew.md [hunk 4]
```diff
@@ -0,0 +1,43 @@
+#<b>Important updates</b>
+
+##v1.1.0 
+
+###Configuration of ‘maven’ builder has changed.
```
Comment (tal-sapan) [inline docs/docs/whatsnew.md:5]: Add a space after the `#` and fix the apostrophe
```suggestion
### Configuration of `maven` builder has changed. 
```

**Review by tal-sapan — APPROVED** [review]

**Pull request merged**